modules/entity_version_workflows/tests/src/Kernel/EntityVersionWorkflowsTest.php

```diff
@@ -79,12 +72,12 @@ public function testEntityVersionWorkflow() {
       'type' => 'entity_version_workflows_example',
       'moderation_state' => 'draft',
     ];
-    /** @var \Drupal\node\NodeInterface $node */
-    $node = $this->nodeStorage->create($values);
+
+    $node = Node::create($values);
     $node->save();
 
     // There is no default value so all versions should be 0.
-    $this->assertEqual('draft', $node->moderation_state->value);
+    $this->assertEquals('draft', $node->moderation_state->value);
```

Comment:
This can be `::assertSame()` (which stand for === comparison, while assertEquals is ==).